Sources/PKYouboraAdsAdapter.swift

```diff
@@ -214,18 +251,19 @@ extension PKYouboraAdsAdapter {
                     guard let self = self else { return }
                     self.lastReportedResource = event.adTagUrl
                 }
-            // When ad request the content to resume (finished or error)
+                // When ad request the content to resume (finished or error)
             // Make sure to send /adStop event and clear the info.
             case let e where e.self == AdEvent.adDidRequestContentResume:
                 messageBus.addObserver(self, events: [e.self]) { [weak self] event in
                     guard let self = self else { return }
                     self.fireStop()
+                    self.tryAdBreakStop()
```

Comment:
what is the purpose of `adBreakStop`  with `firstStop`.
Also, why do we have `fireStop` here? we should have it in completed or error cases already.